### PR TITLE
Removed "?" GitHub link. Replaced it.

### PR DESCRIPTION
**Pull request by saarthakchats**

### CODE REVIEW

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Provide a test link @saarthakchats 
Also, squash the commits! :+1:

File: _includes/footer.html [hunk 1]
```diff
@@ -37,10 +37,10 @@
                                 <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
-                                <li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
+                                <li><a target="_self" href="https://github.com/fossasia"><i class="icon social_linkedin icon-large"></i></a></li>
```
Comment (abishekvashok) [inline _includes/footer.html]: We don't need icon-large do we

**Comment by supunTE** [comment]
Please add more details about what you did

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Also pls squash your commits. If you can't find a good squashing tutorial, query up in the gitter channel.

**Review by nzec — CHANGES_REQUESTED** [review]
Rather than removing it, you could have just put a Github image or linked FontAwesome.

**Comment by mariobehling** [comment]
Please provide add a Github icon.

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@saarthakchats  Please do the suggested changes 👍

**Comment by ParthS007** [comment]
@saarthakchats  PLease provide a Preview link also 👍

**Pull request closed without merging**